x/profiles/keeper/alias_functions.go

```diff
@@ -131,6 +133,41 @@ func (k Keeper) GetApplicationLinks(ctx sdk.Context) []types.ApplicationLink {
 	return links
 }
 
+// IterateExpiringApplicationLinks iterates through all the expiring application links references.
+// The key will be skipped and deleted if the application link has already been deleted.
+func (k Keeper) IterateExpiringApplicationLinks(ctx sdk.Context, fn func(index int64, link types.ApplicationLink) (stop bool)) {
+	store := ctx.KVStore(k.storeKey)
+
+	iterator := sdk.KVStorePrefixIterator(store, types.ExpiringAppLinkTimePrefix)
+	defer iterator.Close()
+
+	i := int64(0)
+	for ; iterator.Valid(); iterator.Next() {
+		trimmedPrefixKey := bytes.TrimPrefix(iterator.Key(), types.ExpiringAppLinkTimePrefix)
```

Comment:
Can you add a comment to describe `Key` and `Value` of iterator representing here?
It would be helpful to understand this closure.

Reply:
Sure, added!